agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/sampling/TraceIdBasedSampler.java

```diff
@@ -20,7 +20,7 @@
 
     private static final Logger logger = LoggerFactory.getLogger(TraceIdBasedSampler.class);
 
-    private static final AttributeKey<Double> AI_SAMPLING_PERCENTAGE = AttributeKey.doubleKey("ai.internal.sampling.percentage");
+    private static final AttributeKey<Double> AI_SAMPLING_PERCENTAGE = AttributeKey.doubleKey("applicationinsights.internal.sampling_percentage");
```

Comment:
can we keep "sampling.percentage"?

Reply:
Look like AI_SAMPLING_PERCENTAGE is no longer used.

Reply:
this is otel convention, dots to separate namespaces, and underscores to separate words

`AI_SAMPLING_PERCENTAGE` is still used below